.circleci/config.yml

```diff
@@ -2,6 +2,7 @@ version: 2.1
 
 orbs:
   python: circleci/python@2.0.3
+  ship: auth0/ship@dev:alpha
```

Comment:
Will update this version later once the Python bits have actually been released in the orb.